cfme/web_ui/__init__.py

```diff
@@ -2322,7 +2322,10 @@ def exists(self):
 
     @property
     def a_cond(self):
-        return "@title={name} or @data-original-title={name}".format(name=quoteattr(self._name))
+        if (self.qtype == "middleware"):
+            return "contains(@title, {name})".format(name=quoteattr(self._name))
```

Comment:
Can you do the check a bit more specifically? eg. put it like:
```python
quoteattr('Name: {}'.format(self._name))  # Or with whatever it is prefixed.
```

This would help reducing the eventual errors from matching multiple quadicons.

And in this case I would also recommend to apply normalize_space(@title), just to be safe with blanks.

Your opinion @psav ?